src/bionics.h

```diff
@@ -42,11 +45,13 @@ struct bionic : public JsonSerializer, public JsonDeserializer {
     int         charge  = 0;
     char        invlet  = 'a';
     bool        powered = false;
+    // this map is stored because JSON definition of bionic can has multiple options
+    std::map<body_part, int> used_bodyparts;
```

Comment:
Consider `std::unordered_map` unless you need to traverse `used_bodyparts` in a specific order